en/app.ftl

```diff
@@ -460,11 +462,36 @@ onboarding-premium-domain-description = With a custom domain, you can make your
 onboarding-premium-reply-title = Reply to emails without giving away your real address
 onboarding-premium-reply-description = Need to reply to emails sent to an alias? Just reply as normal — your alias will still protect your email address.
 
+## Multi-part Premium Onboarding - This is a walk-through onboarding expereince
+multi-part-onboarding-premium-welcome-headline = Welcome to { -brand-name-relay-premium }
+multi-part-onboarding-premium-welcome-subheadline = Now you can control what hits your inbox, one email at a time.
+multi-part-onboarding-premium-welcome-title = Control what emails you get:
+multi-part-onboarding-premium-welcome-description = { onboarding-premium-control-description }
+multi-part-onboarding-premium-domain-headline = Custom domain for sharing aliases
+multi-part-onboarding-premium-domain-subheadline = { multi-part-onboarding-premium-welcome-subheadline }
```

Comment:
This is reused from above. This can be removed in favor of just using the same ID across all three "steps"

Reply:
I removed these and used the original strings.